src/views/v2/list/nav/next.php

```diff
@@ -22,6 +22,19 @@
 		class="tribe-events-c-nav__next tribe-common-b2"
 		data-js="tribe-events-view-link"
 	>
-		<?php echo esc_html( sprintf( __( 'Next %s', 'the-events-calendar' ), tribe_get_event_label_plural() ) ); ?>
+		<?php
+			echo sprintf(
+				wp_kses(
+					/* translators: %s: Event (plural or singular). */
+					__( 'Next<span class="tribe-events-c-nav__next--label-plural"> %s</span>', 'the-events-calendar' ),
+					array(
+						'span' => array(
+							'class' => array(),
+						),
+					)
+				),
+				tribe_get_event_label_plural()
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: b0380c6b6aa818840a4208921941c81a</sub>